src/modules/tests/tests.service.ts

```diff
@@ -463,4 +464,147 @@ export class TestsService {
     };
   }
 
+
+  async getUserTestResult(testId: string, userId: string, authContext: AuthContext): Promise<{
+    testId: string;
```

Comment:
@vinayak-patil can all these fields come from a file where you define a costant var?